### PR TITLE
Fix logging big.Int value

### PR DESCRIPTION
**Pull request by hiyosi**
Signed-off-by: Tomoya Usami <tousami@zlab.co.jp>

**Pull Request check list**

- [x] Commit conforms to CONTRIBUTING.md?
- [x] Proper tests/regressions included?
- [ ] Documentation updated?

**Affected functionality**

SPIRE Server middleware

**Description of change**

This PR fixes the broken logging.
Before this commit, the bit.Int wasn't properly output.

```
 time="2020-10-16T07:43:08Z" level=error msg="Failed to fetch authorized entries" error="rpc error: code = PermissionDenied desc = agent \"spiffe://example.org/spire/agent/sshpop/${hostname}\" expected to have serial number \"173519580944580228740697109256257631213\"; has %!q(big.Int=107959714025526768817333596538168335315)" subsystem_name=manager
```

**Which issue this PR fixes**

### CODE REVIEW

**Review by azdagron — DISMISSED** [review]
Huh, TIL big.Int's implementation of `fmt.Formatter` doesn't handle `%q`. That seems like an omission.

In any case, thanks!

**Comment by azdagron** [comment]
I've opened up an issue on Go. https://github.com/golang/go/issues/42022.

**Comment by azdagron** [comment]
@hiyosi when you have a chance can you update the branch so we can merge this? I'm also happy to do it if you want to click the "Allow edits from maintainers" check box.

**Comment by hiyosi** [comment]
@azdagron Thanks!

When the source repository of a PR is belong to an organization, it seems that the option is disabled.
I'll consider to change the source repository in the next time.

**Pull request merged**
